dlk/tests/test_code_generation.py

```diff
@@ -389,15 +389,15 @@ def test_code_generation(self, i, configuration) -> None:
         self.run_test_all_configuration(i, configuration)
 
 
-class TestCodeGenerationArm(TestCodeGenerationBase):
+class TestCodeGenerationArm(TestCodeGenerationBase, TestCaseFPGABase):
```

Comment:
I haven't checked these base classes, but it is better to avoid multiple inheritance because it adds complexity. Is there a chance we could merge them?

Reply:
@tsawada Thanks for reviewing!
OK, I see. I changed them to use a class variable `setup_fpga` as a flag for checking whether the setup is needed instead of multiple inheritances with splitting class.
Please review it again! 🙇